environment.conf.e2e.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Is this file necessary? It's yet another copy of the whole configuration that needs to be maintained.

Reply:
I agree, not a fan of this solution as well. When need to merge #1094 in order to be able to inherit the base configuration.

Reply:
I think we can remove this file now, right?